package.json

```diff
@@ -202,21 +190,21 @@
     },
     {
       "author": "whyrusleeping",
-      "hash": "QmbHGPnENTUvxBsidRZasYYTFT2DJWYHLpAzcpssSxZupj",
+      "hash": "QmQJpT1PrAgiZcPS93hFNJS5Zc355oNYfu5vfZKDnyyBpc",
       "name": "go-ds-flatfs",
-      "version": "1.1.7"
```

Comment:
Can we please wait on updating this dependency.  I would like to get https://github.com/ipfs/go-ds-flatfs/pull/35 in first.  Otherwise we will have an on-disk format change.